### PR TITLE
✨ Mobile | Fix the answer field for quizzes

### PR DESCRIPTION
**Pull request by AntPolkanov**
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

✏️ The input box for quizzes was single line. Fixes the "OS keyboard and editor (enable multiline)" task on #615 

> 2. What was changed?

✏️ 
The input box is now multiline
The input box is transparent, hence the page feels lighter
Removed underline from the input box on Android
The cursor color is Red
The keyboard hides when the user clicks outside of it on the PeoplePage
Upgraded MAUI to the latest version
Replaced `ICommand` with `RelayCommand`

**Figure 1:** This is how the multiline input box looks like now.

**Figure 2:** The multiline input box scales up and down when user adds or deletes text.

**Figure 3:** The keyboard has a Done button at the right top corner of the iOS keyboard.

> 3. Did you do pair or mob programming?

✏️ No

### CODE REVIEW

File: src/MobileUI/Pages/EarnDetailsPage.xaml.cs
Comment (matt-goldman) [inline src/MobileUI/Pages/EarnDetailsPage.xaml.cs:1]: We shoud use `EventToCommandBehavior` for this.

File: src/MobileUI/ViewModels/EarnDetailsViewModel.cs
Comment (matt-goldman) [inline src/MobileUI/ViewModels/EarnDetailsViewModel.cs:1]: We should update all the `ICommand` instances to use `[RelayCommand]` from the MVVM CommnunityToolkit. @zacharykeeping already did this for the `ObservableProperty` instances.

See: https://github.com/SSWConsulting/SSW.Rewards.Mobile/issues/711

(You can mark this as closed - not blocking this PR on this comment)

**Review by matt-goldman — CHANGES_REQUESTED** [review]
Looks good, although please use an event to command behavior (there's one ready-rolled in the MAUI Community Toolkit, which I think we're already using elsewhere) instead of event handlers.

File: src/MobileUI/ViewModels/EarnDetailsViewModel.cs
Comment (AntPolkanov) [reply src/MobileUI/ViewModels/EarnDetailsViewModel.cs:1]: Done

File: src/MobileUI/Pages/EarnDetailsPage.xaml.cs
Comment (AntPolkanov) [reply src/MobileUI/Pages/EarnDetailsPage.xaml.cs:1]: Done

**Review by matt-goldman — APPROVED** [review]
🚀 LGTM - I think we should review whether that handler should be in `MauiProgram` (I don't they generally should be, although this one might be alright as it solves a problem). I think we have other handlers in other places. I've added this issue for that: https://github.com/SSWConsulting/SSW.Rewards.Mobile/issues/713

**Pull request merged**
